content/en/docs/collector/configuration.md

```diff
@@ -543,3 +547,43 @@ service:
       exporters:
         - otlp/auth
 ```
+
+
+### Setting up certificates
+
+Certain exporters require that x.509 certificates are used for mTLS security.
```

Comment:
How about something like this?

```suggestion
For a production setup, we strongly recommend using TLS certificates, either for secure communication or mTLS for mutual authentication.
```

Reply:
I like your suggested change @jpkrohling.